frontend/src/components/MultiSelect.js

```diff
@@ -57,6 +57,21 @@ function MultiSelect({
   onCreateOption,
   components: componentReplacements,
 }) {
+  const inputId = `select-${uuidv4()}`;
+
+  useEffect(() => {
+    /**
+     * unfortunately, given our support for ie11, we can't
```

Comment:
Something we need to keep an eye on, ie11 end of life is June 2022. We also can't update `react-hook-form` because it drops ie11 support. I wish we had analytics on how many users actually use ie11.

Reply:
Yes - I know we've talked about adding our own analytics/recording audit info as an upcoming feature, I definitely want to record user agent when we do so (unless we can get it from New Relic somehow)